src/test/java/com/endava/internship/collections/StudentSetTest.java

```diff
@@ -0,0 +1,241 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.*;
+import java.time.LocalDate;
+
+public class StudentSetTest {
+    StudentSet set;
+
+
+    @BeforeEach
+    void setUpStudentSet(){
+        set = new StudentSet();
+    }
+
+    @Test
+    void addElementShouldReturnTrue(){
+        Student st = new Student("John Doe", LocalDate.of(2000, 5, 12), "");
+        assertTrue(set.add(st));
```

Comment:
assert colection is not empty and size = 1

Reply:
Fixed test